net-news/yencode/yencode-0.46-r1.ebuild

```diff
@@ -12,26 +14,25 @@ IUSE="nls"
 
 DEPEND=""
 
-src_compile() {
+src_configure() {
 	local myconf
```

Comment:
Please turn `myconf` into an array, and include all options in it to avoid the backslash hell.